worker/src/lib/dbLogger/DBLoggable.ts

```diff
@@ -715,13 +719,15 @@ export class DBLoggable {
           experimentRowIndex:
             requestHeaders.experimentHeaders.rowIndex ?? undefined,
         },
+
         response: {
           id: this.response.responseId,
           status: await this.response.status(),
           bodySize: rawResponseBody.length,
           timeToFirstToken,
           responseCreatedAt: endTime,
-          delayMs: endTime.getTime() - this.timing.startTime.getTime(),
+          delayMs: cacheReferenceId == DEFAULT_UUID ? endTime.getTime() - this.timing.startTime.getTime() : 0,
+          cachedLatency: cacheReferenceId == DEFAULT_UUID ? 0 : Number(cachedHeaders?.get("Helicone-Cache-Latency")) ?? 0,
```

Comment:
```suggestion
          cachedLatency: cacheReferenceId == DEFAULT_UUID ? 0 : Number(cachedHeaders?.get("Helicone-Cache-Latency")) ?? 0,
```

Can we make this safer, put it in a helper function around a try catch or something plz